runtime/gc_glue_java/configure_includes/configure_linux_390.mk

```diff
@@ -30,6 +30,7 @@ endif
 CONFIGURE_ARGS += \
 	--enable-OMR_THR_THREE_TIER_LOCKING \
 	--enable-OMR_THR_YIELD_ALG \
+	--enable-OMR_GC_DOUBLE_MAP_ARRAYLETS \
```

Comment:
I am not sure how it should be exactly but this looks wrong because define this flag for both 32 and 64 bit platforms with intention to be on 64 bit platforms only... this apply to all places like this